### PR TITLE
Add explanation of reporting vulnerabilities

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Supported Versions
+
+| Version   | Supported          |
+|-----------|--------------------|
+| 1.16.x    | :white_check_mark: |
+| <= 1.15.x | :x:                |
+
+## Reporting a Vulnerability
+
+If you found the vulnerability about default configuration of **fluentd-daemonset-SOMETHING**, report from the following page:
```
Comment (daipom) [inline SECURITY.md:12]: How about this?

```suggestion
If you find a vulnerability of **fluentd-daemonset-SOMETHING** with the default configuration, report it from the following page:
```

Comment (kenhys) [reply SECURITY.md]: Thanks!

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+| <= 1.15.x | :x:                |
+
+## Reporting a Vulnerability
+
+If you find a vulnerability of **fluentd-daemonset-SOMETHING** with the default configuration, report it from the following page:
+
+* https://github.com/fluent/fluentd/security/advisories
+
+> [!IMPORTANT]
+> As fluentd-kubernetes-daemonset images are downstream of [fluent/fluentd](https://hub.docker.com/r/fluent/fluentd) container.
```
Comment (daipom) [inline SECURITY.md:17]: I believe this 'As' is not necessary.

```suggestion
> fluentd-kubernetes-daemonset images are downstream of [fluent/fluentd](https://hub.docker.com/r/fluent/fluentd) container.
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+> Thus, even though security scanner reports pile of vulnerabilities, the updated container image can't be shipped
```
Comment (daipom) [inline SECURITY.md:18]:
```suggestion
> Thus, even though security scanner reports a pile of vulnerabilities, the updated container image can't be shipped
```

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+> until updated container image is deployed from upstream at first.
```
Comment (daipom) [inline SECURITY.md:19]: `at first` doesn't sound right in this context.

```suggestion
> until updated container image is deployed from upstream first.
```

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+* If you found that bundled Ruby gems which are related to **fluentd-daemonset-SOMETHING** have vulnerabilities, please report to [fluentd-kubernetes-daemonset](https://github.com/fluent/fluentd-kubernetes-daemonset/issues/new).
+
+* If you found that bundled Ruby gems which are related to **fluentd-docker-image** have vulnerabilities, please report to [fluentd-docker-image](https://github.com/fluent/fluentd-docker-image/issues/new).
```
Comment (daipom) [inline SECURITY.md:23]: `which are` seems to be misleading. 
(`that`に比べると単なる付加情報を足すものらしいです、ぼくもよくわかりませんが、、省略できそうなので省略しちゃったほうがよさそうだなと思いました)

```suggestion
* If you find that bundled Ruby gems related to **fluentd-daemonset-SOMETHING** have vulnerabilities, please report to [fluentd-kubernetes-daemonset](https://github.com/fluent/fluentd-kubernetes-daemonset/issues/new).

* If you find that bundled Ruby gems related to **fluentd-docker-image** have vulnerabilities, please report to [fluentd-docker-image](https://github.com/fluent/fluentd-docker-image/issues/new).
```

**Review by daipom — APPROVED** [review]
Thanks!
Basically looks good to me!
I have commented on some points I am concerned about expressing.
I'll leave it to you to apply it or not.

**Comment by kenhys** [comment]
rebased feedback.

**Pull request merged**